src/ui/mobile.js

```
@@ -982,3 +982,18 @@ export function handleTouchEnd(e) {
     }
   }
 }
+
+// Setter functions for virtual keys
+export function setVirtualKey(key, value) {
+  if (virtualKeys.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)